core/chains/evm/logpoller/log_poller.go

```diff
@@ -104,7 +105,8 @@ type ReplayRequest struct {
 // support chain, polygon, which has 2s block times, we need RPCs roughly with <= 500ms latency
 func NewLogPoller(orm *ORM, ec Client, lggr logger.Logger, pollPeriod time.Duration,
 	finalityDepth int64, backfillBatchSize int64, rpcBatchSize int64, keepBlocksDepth int64) *logPoller {
-	return &logPoller{
```

Comment:
why this change? generally go advises against defining a short var like x := &s since it masks the pointer nature of the var hurting readability (usually preferable to & at stack frame boundaries to make it clear its returning a pointer)

Reply:
Not intentional, I probably changed it while debugging or refactoring, and forgot to put it back.  Hadn't heard that about preferring & at stack boundary thanks for the tip.